packages/eslint-config-ckeditor5/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-ckeditor5",
-  "version": "6.0.0",
+  "version": "7.0.0",
```

```suggestion
  "version": "6.0.0",
```

Release scripts should do this, so please revert it. Devops will handle it.